### PR TITLE
xds: Blank import supported certificate provider plugins.

### PR DESCRIPTION
**Pull request by easwars**
This is required to make sure that PSM security works seamlessly.
Without this, the user would have to do the blank import of all plugins
which they expect their control plane to send down.

### CODE REVIEW

File: xds/go13.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package xds
+
+import (
+	_ "google.golang.org/grpc/credentials/tls/certprovider/meshca" // Register the MeshCA certificate provider plugin.
```
Comment (dfawley) [inline xds/go13.go:24]: Should we move this back after #4051?

File: examples/go.sum [hunk 2]
```diff
@@ -27,6 +27,7 @@ github.com/google/go-cmp v0.4.0 h1:xsAVV57WRhGj6kEIi8ReJzQlHHqcBYCElAvkovg3B/4=
 github.com/google/go-cmp v0.4.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
 github.com/google/go-cmp v0.5.0 h1:/QaMHBdZ26BB3SSst0Iwl10Epc+xhTquomWX0oZEB6w=
 github.com/google/go-cmp v0.5.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
+github.com/google/uuid v1.1.2 h1:EVhdT+1Kseyi1/pUmXKaFxYsDNy9RQYkMWRH68J/W7Y=
```
Comment (dfawley) [inline examples/go.sum:30]: Revert?

Comment (easwars) [reply examples/go.sum]: Done.

Comment (easwars) [reply examples/go.sum:30]: Actually vet isnt happy without this. So, re-reverted it.

File: xds/go13.go (hunk 1 above)
Comment (dfawley) [reply xds/go13.go:24]: The xds interop should, but not the non-xds interop client.

https://github.com/grpc/grpc-go/blob/master/interop/client/client.go
vs
https://github.com/grpc/grpc-go/blob/master/interop/xds/client/client.go

I don't know what Go runtime we use for the xds tests, but I suspect it's more modern.  @ericgribkoff do you know the answer to this?

Comment (menghanl) [reply xds/go13.go:24]: It's what `apt get` gives us. Seems to be `go version go1.11.6 linux/amd64`

Comment (dfawley) [reply xds/go13.go:24]: Can this be changed?

File: xds/go13.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// +build go1.13
```
Comment (dfawley) [inline xds/go13.go:1]: Nit: filename should be `go113.go` to match our more commonly-used convention (there is one `go12.go`).

**Review by dfawley — APPROVED** [review]
Approving.  I'm fine with this approach for now given the current state of the xds interop tests.

Please fix the nit before merging.

File: xds/go13.go (hunk 3 above)
Comment (easwars) [reply xds/go13.go]: Done.

**Pull request merged**